src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { TableLink } from './TableLink';
+
+const EMPTY_FIELD = '-';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { person } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
Create constant for column names and use `map` here

Reply:
Not fixed 😢